test/ngResource/resourceSpec.js

```diff
@@ -750,6 +750,7 @@ describe('basic usage', function() {
     var json = JSON.parse(angular.toJson(cc));
     expect(json.$promise).not.toBeDefined();
     expect(json.$resolved).not.toBeDefined();
+    expect(json.$cancelRequest).not.toBeDefined();
```

Comment:
Can you also add an expectation that is was there before? (Similar to what we have for `$promise`/`$resolved`.)